homeassistant/brands/philips.json

```diff
@@ -1,5 +1,5 @@
 {
   "domain": "philips",
   "name": "Philips",
-  "integrations": ["dynalite", "hue", "philips_js"]
+  "integrations": ["dynalite", "hue", "philips_js", "philips_airpurifier_coap"]
```

Comment:
We don't add custom integrations in these brands files. There's no direct support for custom integrations in core, besides the general features that allow them to be run in core.